lib/mvi/responses/id_parser.rb

```diff
@@ -31,7 +31,7 @@ def parse(ids)
                                          %w[H L])&.first,
           vha_facility_ids: select_facilities(select_extension(ids, /^\w+\^PI\^\w+\^USVHA\^\w+$/, CORRELATION_ROOT_ID)),
           birls_id: select_ids(select_extension(ids, /^\w+\^PI\^200BRLS\^USVBA\^\w+$/, CORRELATION_ROOT_ID))&.first,
-          vet360_id: select_ids(select_extension(ids, /^\w+\^PI\^200VETS\^USDVA\^\w+$/, CORRELATION_ROOT_ID))&.first,
+          vet360_id: select_ids(select_extension(ids, /^\w+\^PI\^200VETS\^USDVA\^A$/, CORRELATION_ROOT_ID))&.first,
```

Comment:
There can be historical Vet360 IDs.  Adding the `A` flag requirement to ensure only an active ID is selected.